### PR TITLE
Elastio with PHP Lumen API

### PR DESCRIPTION
**Pull request by rsaylor73**

### CODE REVIEW

**Review by anelson — CHANGES_REQUESTED** [review]
This is a great example of how Elastio can be integrated into another application.

PHP isn't a language I have any experience with so I don't feel qualified to review the PHP code; we'll accept that as-is.  However I've made a few comments about the interaction with `elastio`.  I placed them on the EC2 backup code but they apply to all of the `elastio` invocations.

File: elastio-api-php-lumen/app/Service/EapService.php [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+    /**
+     * @param $json
+     * @return JsonResponse
+     */
+    public static function newEc2Backup($json): JsonResponse
+    {
+        $instanceID = $json['instanceID'];
+
+        $cmd = "cd /home/ubuntu && sudo elastio ec2 backup --instance-id {$instanceID} 2>&1; echo $?";
```
[the remark this answers is not in the document]
Comment (rsaylor73) [reply elastio-api-php-lumen/app/Service/EapService.php:108]: yes the API is running as a web user "www-data" if on Ubuntu so we need to be in the visuders file that is documented. 

Another option would be to queue the job then run from cron but with this simple integration that was not the direction.

The path does not exist when running from a php shell. From past experience this is the way.

[the remark this answers is not in the document]
Comment (rsaylor73) [reply elastio-api-php-lumen/app/Service/EapService.php:114]: Added sanitation on all inputs from the user.

**Review by anelson — APPROVED** [review]
:+1:

**Pull request merged**